### PR TITLE
Fix some items' names for Alberta stylish npc

### PR DESCRIPTION
**Pull request by austin-sa-wang**
NPC: `'Bao Bao', 'Crescent Hairpin', 'Fashionable Glasses', 'Heart Hairpin' Quest`

Fixed names to match db's english name

* **Addressed Issue(s)**: 
NPC: `'Bao Bao', 'Crescent Hairpin', 'Fashionable Glasses', 'Heart Hairpin' Quest`

Fixed names to match db's english name

* **Server Mode**: Both

* **Description of Pull Request**:  Just updating strings

### CODE REVIEW

File: npc/quests/quests_alberta.txt [hunk 1]
```diff
@@ -782,7 +782,7 @@ alberta,136,79,1	script	Hat store girl#new30	71,{
 							next;
 							delitem 2252,1; //Star_Sparkling
 							delitem 1036,400; //Dragon_Scale
-							delitem 4052,1; //Elder_Wilow_Card
+							delitem 4052,1; //Elder_Willow_Card
```
Comment (secretdataz) [inline npc/quests/quests_alberta.txt:785]: This is AegisName reference which contains the typo.

File: npc/quests/quests_alberta.txt [hunk 2]
```diff
@@ -617,7 +617,7 @@ alberta,120,53,1	script	Stylish Merchant#new30	51,{
 			next;
 			mes "[Zic]";
 			mes "Okay, I'll need...";
-			mes "1 ^0000FFHeart hair pin^000000";
+			mes "1 ^0000FFHeart hairpin^000000";
```
Comment (secretdataz) [inline npc/quests/quests_alberta.txt:620]:
```suggestion
			mes "1 ^0000FFHeart Hairpin^000000";
```

**Review by secretdataz — CHANGES_REQUESTED** [review]
Looks good overall.

File: npc/quests/quests_alberta.txt (hunk 1 above)
Comment (Lemongrass3110) [inline npc/quests/quests_alberta.txt:785]:
```suggestion
							delitem 4052,1; //Elder_Wilow_Card
```

**Review by Lemongrass3110 — APPROVED** [review]

**Pull request merged**